src/nwbinspector/_dandi_inspection.py

```diff
@@ -21,7 +21,7 @@ def inspect_dandiset(
     importance_threshold: Union[str, Importance] = Importance.BEST_PRACTICE_SUGGESTION,
     skip_validate: bool = False,
     show_progress_bar: bool = True,
-    client: Union["dandi.dandiapi.DandiAPIClient", None] = None,
+    client: Union["dandi.dandiapi.DandiAPIClient", None] = None,  # type: ignore
```

Comment:
Unfortunately does not play well with lazy imports, so those are flagged as skips